packages/adp-tooling/package.json

```diff
@@ -44,7 +44,7 @@
         "mem-fs": "2.1.0",
         "mem-fs-editor": "9.4.0",
         "prompts": "2.4.2",
-        "yazl": "2.5.1"
+        "adm-zip": "0.5.10"
```

Comment:
Blocking the merge by myself now :)
Something in my manual local tests is wrong with the zip file. Not sure if it is just me or something is indeed wrong

Reply:
@longieirl @zdravko-georgiev please re-review. I found an issue with the deploy flow from CLI which I had not properly manually tested before. It is fixed now, and I have added a unit test catching the issue.